console

```diff
@@ -16,6 +20,29 @@ $application = require __DIR__ . '/res/bootstrap.php';
 
 $cli = $application->cli();
 
+$cli->command('serve', function (\Stratify\Http\Application $http, ContainerInterface $container, OutputInterface $output) {
+    $loop = React\EventLoop\Factory::create();
+
+    $db = $container->get(Connection::class);
+
+    $server = new Server(function (ServerRequestInterface $request) use ($http, $db) : ResponseInterface {
+        // Reset the DB connection
+        $db->close();
+        return $http->handle($request);
+    });
+    $server->listen(new React\Socket\Server($container->get('http.port'), $loop));
+    $server->on('error', function (Exception $e) use ($output) {
+        $output->writeln("<error>Error: {$e->getMessage()}</error>");
+        if ($e->getPrevious() !== null) {
+            $e = $e->getPrevious();
+            $output->writeln("<error>Caused by: {$e->getMessage()}</error>");
+        }
+        $output->writeln($e->getTraceAsString());
+    });
+
+    $loop->run();
+});
```

Comment:
The `./console serve` command runs the React webserver.

Since React supports PSR-7 this is extremely easy to integrate with Stratify or any other PSR-7 framework.